packages/shared/src/hooks/useFeed.ts

```diff
@@ -235,5 +236,6 @@ export default function useFeed<T>(
     canFetchMore: feedQuery.hasNextPage,
     emptyFeed:
       !feedQuery?.data?.pages[0]?.page.edges.length && !feedQuery.isFetching,
+    isLoading: feedQuery.isLoading,
```

Comment:
Exposed `isLoading` from hook. Needed it to check initial load state in the `Feed`. This is not breaking anything else.